apps/fabric-website/src/pages/HomePage/HomePage.styles.ts

```diff
@@ -398,9 +398,9 @@ export const getStyles = (props: IHomePageStyleProps): IHomePageStyles => {
         color: 'inherit',
 
         selectors: {
-          '&:hover, &:active, &:active:hover': {
+          '&:hover, &:active, &:active:hover, &:link': {
             textDecoration: 'none',
-            color: 'inherit'
+            color: isInverted ? palette.white : palette.black
```

Comment:
@jordandrako Wanted to explicitly call this out for you to look at since I added it after you approved--this is what I'm trying to fix.

![image](https://user-images.githubusercontent.com/5864305/57752452-1a51f580-769e-11e9-914f-7f7cab233fa0.png)

Reply:
Yeah, that change is probably a good call. This style regression is yet again caused by the UHF global styles.

Reply:
Actually, at second glance, I'm not sure this will target the link correctly. Keep the `&:link` part on line 401, but `isInverted ? palette.white : palette.black` probably have to be on line 398 instead of 403, and leave 403 as `inherit`.

Reply:
Just tested this some more, and I don't think the `link` class is going to target this specific list of links. On line 329, the `card` class targets `'a, a:hover, a:active, p'`. I think that should be changed to:

```tsx
card: [
      classNames.card,
      {
        padding: appPadding.small,
        flex: '0 0 25%',
        minWidth: 300,

        selectors: {
          ['a:not(.is-disabled), a:not(.is-disabled):link,' +
          'a:not(.is-disabled):hover, a:not(.is-disabled):active,' +
          'a:not(.is-disabled):active:hover p']: {
            color: isInverted ? palette.black : palette.white
          },

          [mediaQuery.maxLarge]: {
            flex: '1 0 25%'
          }
        }
      }
    ],
```

Reply:
I should stop reviewing code at nearly midnight...

The above didn't change it in my testing either. Only setting `color` on the parent element does, which in this case is `cardListItem`.

Reply:
Would you mind taking another look at the current code?

Reply:
It seems I also should stop writing code at midnight, because I just realized I reversed the colors for inverted and normal... (now fixed)